dask/array/core.py

```diff
@@ -5549,29 +5557,32 @@ class BlockView:
     An instance of ``da.array.Blockview``
     """
 
-    def __init__(self, array: Array) -> BlockView:
+    def __init__(self, array: Array) -> None:
```

Comment:
Return type of ``__init__`` should be omitted
```suggestion
    def __init__(self, array: Array):
```